test/src/main/docker/garmadon/scripts/entrypoint.sh

```diff
@@ -8,6 +8,10 @@ function check_resourcemanager_up {
     until curl http://resourcemanager:8088/; do sleep 1; done
 }
 
+function check_es_up {
```

Comment:
This change should be in a separate review

Reply:
No I need it now since I load template from ES reader

Reply:
Ok got it, that was for the block_until_website_available that you removed